src/Controller/OrderController.php

```diff
@@ -147,6 +148,13 @@ public function orderCreateAction($participantId, Request $request, SessionInter
                 } elseif ($request->request->has('saliva')) {
                     $order->setType('saliva');
                 }
+                if ($participant->isPediatric && $request->request->has('blood')) {
```

Comment:
Could you separate out the isPediatric check and then check for the sample type?

```suggestion
                if ($participant->isPediatric) {
                     if ($request->request->has('blood')) {
```